can_tools/scrapers/official/DC/dc_vaccines.py

```diff
@@ -17,49 +17,57 @@ class DCVaccineSex(TableauDashboard):
     data_tableau_table = "Demographics "
 
     # map column names into CMUs
-    cmus = {
-        "PARTIALLY VACCINATED": CMU(
-            category="total_vaccine_initiated", measurement="cumulative", unit="people"
-        ),
-        "FULLY VACCINATED": CMU(
-            category="total_vaccine_completed", measurement="cumulative", unit="people"
-        ),
+    variables = {
+        "FULLY VACCINATED": variables.FULLY_VACCINATED_ALL,
+        "INITIATED": variables.INITIATING_VACCINATIONS_ALL,
     }
 
     def _get_date(self):
         # 'last updated' date is stored in a 1x1 df
         df = self.get_tableau_view(
             url=(self.baseurl + "/views/Vaccine_Public/Administration")
-        )["Update"]
+        )["Admin Update"]
         return pd.to_datetime(df.iloc[0]["MaxDate-alias"]).date()
 
     def normalize(self, data):
-        df = (
-            data.rename(
-                columns={
-                    "Vaccination Status-alias": "variable",
-                    "SUM(Vaccinated)-value": "value",
-                }
-            )
-            .assign(
-                vintage=self._retrieve_vintage(),
-                dt=self._get_date(),
-                location=self.state_fips,
-            )
-            .drop(
-                columns={
-                    "SUM(Vaccinated)-alias",
-                    "Cross-alias",
-                }
-            )
+        df = data.rename(
+            columns={
+                "Vaccination Status-alias": "variable",
+                "SUM(Vaccinated)-value": "value",
+                "Cross-value": "demo_val",
+            }
+        ).drop(
+            columns={
+                "SUM(Vaccinated)-alias",
+                "Cross-alias",
+            }
         )
+        df["demo_val"] = df["demo_val"].str.lower()
 
-        # already in long form yay
-        df = df.pipe(self.extract_CMU, cmu=self.cmus)
-        df["value"] = df["value"].astype(int)
+        # check to ensure that the scraper is still returning data by race
+        if not all(x in list(df["demo_val"].unique()) for x in ["white", "black"]):
+            raise ValueError("scraper is not returning race data")
 
-        out = df.assign(sex=df["Cross-value"].str.lower()).drop(
-            columns={"Cross-value", "variable"}
+        # sum the partially and fully vaccinated entries to match definition
+        # to avoid pivoting to wide then back to long I selected each corresponding entry to sum
+        q = 'variable == "{v} VACCINATED" and demo_val == "{d}"'
+        for d in ["black", "white", "other"]:
+            initiated_value = int(
+                df.query(q.format(v="PARTIALLY", d=d))["value"]
+            ) + int(df.query(q.format(v="FULLY", d=d))["value"])
+            row = {"variable": "INITIATED", "demo_val": d, "value": initiated_value}
```

Comment:
Let's also add the "unknown" data here

I think the values for (age, race, ethnicity, sex) would then be (all,unknown,unknown,all)

Reply:
new commit should add this. I had to do some calculations but it looks to be correct